server/models/k8s_context.go

```diff
@@ -329,3 +332,138 @@ func (kc *K8sContext) AssignServerID() error {
 
 	return nil
 }
+
+// FlushMeshSyncData will flush the meshsync data for the passed kubernetes contextID
+func FlushMeshSyncData(ctx context.Context, ctxID string, provider Provider, eb *events.EventStreamer) {
```

Comment:
We are dropping tables with meshsync data in the ` resyncCluster` resolver, only not streaming events (which we should stream). Why duplicate things, instead here https://github.com/meshery/meshery/pull/6435/files#diff-9640f3bd7dcd176aec6e4ee8412922560c18e16cd87c7f1e92ff6795dab9d64eR205
 we should call `FlushMeshSyncData`

Reply:
yeah, good idea. I'll update that as well